pkg/api/customization/kontainerdriver/store.go

```diff
@@ -0,0 +1,57 @@
+package kontainerdriver
+
+import (
+	"fmt"
+
+	errorsutil "github.com/pkg/errors"
+	"github.com/rancher/norman/httperror"
+	"github.com/rancher/norman/types"
+	"github.com/rancher/types/apis/management.cattle.io/v3"
+	"github.com/rancher/types/config"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/client-go/tools/cache"
+)
+
+type store struct {
+	types.Store
+	KontainerDriverLister v3.KontainerDriverLister
+	ClusterIndexer        cache.Indexer
+}
+
+func NewStore(management *config.ScaledContext, s types.Store) types.Store {
```

Comment:
NewStore now uses the unexported store struct and returns an interface to the type.Store defined in Norman.